From e51fbe465a6a6b63bed15e87afdfa266ed06f3de Mon Sep 17 00:00:00 2001
From: Vasyl Vavrychuk <vasyl.vavrychuk@opensynergy.com>
Date: Mon, 25 Apr 2022 10:59:43 +0300
Subject: [PATCH] Remove unused save_termios

Signed-off-by: Vasyl Vavrychuk <vasyl.vavrychuk@opensynergy.com>

---
 psplash-fb.h | 1 -
 psplash.h    | 1 -
 2 files changed, 2 deletions(-)

diff --git a/psplash-fb.h b/psplash-fb.h
index eafa293..6c4599c 100644
--- a/psplash-fb.h
+++ b/psplash-fb.h
@@ -22,7 +22,6 @@ typedef struct PSplashFB
 {
   int            fd;
   struct fb_var_screeninfo fb_var;
-  struct termios save_termios;
   int            type;
   int            visual;
   int            width, height;
diff --git a/psplash.h b/psplash.h
index e8b6660..02d6f32 100644
--- a/psplash.h
+++ b/psplash.h
@@ -32,7 +32,6 @@
 #include <sys/stat.h>
 #include <sys/time.h>
 #include <sys/types.h>
-#include <termios.h>
 #include <unistd.h>
 #include <stdbool.h>
 
